src/models/constructor-items/common.ts

```diff
@@ -338,6 +344,20 @@ export interface DataLensObjectProps {
     theme: 'dark' | 'light';
 }
 
+export interface IframeProps {
```

Comment:
which of this props we suppose to get from content configs?

Reply:
I thought that it could be all of this props